packages/flutter/lib/src/material/banner.dart

```diff
@@ -120,6 +123,16 @@ class MaterialBanner extends StatefulWidget {
   /// Typically this is a list of [TextButton] widgets.
   final List<Widget> actions;
 
+  /// The z-coordinate at which to place the material banner.
+  /// This controls the size of the shadow below the material banner.
```

Comment:
Nit: We always put a new line between the first sentence and any other documentation for a property
```suggestion
  ///
  /// This controls the size of the shadow below the material banner.
```